test.py

```diff
@@ -582,21 +582,21 @@ class SomeModel(pw.Model):
       some_field = pw.DateTimeField()
       class Meta:
         database = self.db
-    self.evolve_and_check_noop()
+    self.evolve_and_check_changes()
     peeweedbevolve.clear()
     class SomeModel(pw.Model):
       some_field = pwe.DateTimeTZField()
       class Meta:
         database = self.db
-    self.evolve_and_check_noop()
+    self.evolve_and_check_changes([('ALTER TABLE "somemodel" ALTER "some_field" TYPE TIMESTAMPTZ', [])])
```

Comment:
This is the only test changed, I just renamed the `evolve_and_check_noop` function to allow comparing to actual changes.

This behavioral change occurred because peewee previously set the type to `datetime_tz` but now sets it to `timestamptz`. (Herman should perhaps be updated to match this so that herman passes this test.)